GitHub-Test-3/Gardiner_GitTest3.md

```diff
@@ -0,0 +1,11 @@
+# Joey Gardiner Git Test 3
+
+This is markdown! I can write something here such as, I like Star Wars.
+Or I can write it in an in-line code block, like this: `<xml attribute="name"></xml>`.
+
+Or a whole entire code block like this, I think:
+``` 
+This is definitely code.
+```
```

Comment:
@jbg5721 Good work with markdown! This is an example of a **line comment** written directly on a commit to a remote GitHub repo, and it is *also* written in markdown. You'll be writing one of these line comments on another student's commit + Pull Request for the last portion of the Git/GitHub test.